app/logicalOperators.js

```diff
@@ -2,10 +2,10 @@ exports = typeof window === 'undefined' ? global : window;
 
 exports.logicalOperatorsAnswers = {
   or: function(a, b) {
-
+      return a > 0 || b > 0;
   },
 
   and: function(a, b) {
-
+      return a > 0 & b > 0;
```

Comment:
this is incorrect:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_AND
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND

(but test apparently is not designed properly to get the difference :) )